src/ray/gcs/redis_context.cc

```diff
@@ -282,22 +282,38 @@ void RedisContext::Disconnect() {
   redis_async_context_.reset();
 }
 
-Status AuthenticateRedis(redisContext *context, const std::string &password) {
+Status AuthenticateRedis(redisContext *context,
+                         const std::string &username,
+                         const std::string &password) {
   if (password == "") {
     return Status::OK();
```

Comment:
if password is empty, we can `RAY_CHECK(username.empty());` ?

Reply:
Yes, we can.